<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>787</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>1011</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 240, 240);
border: 2px solid #800000;
border-radius: 6px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTabWidget" name="DataTable">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>981</width>
       <height>581</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>4</number>
     </property>
     <widget class="QWidget" name="Tenant">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <widget class="QTableView" name="TenantTable">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>951</width>
         <height>531</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Room">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <widget class="QTableView" name="RoomTable">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>951</width>
         <height>531</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Rent">
      <attribute name="title">
       <string>Page</string>
      </attribute>
      <widget class="QTableView" name="RentTable">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>951</width>
         <height>531</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Payment">
      <attribute name="title">
       <string>Page</string>
      </attribute>
      <widget class="QTableView" name="PaymentTable">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>951</width>
         <height>531</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="EmergencyContact">
      <attribute name="title">
       <string>Page</string>
      </attribute>
      <widget class="QTableView" name="ECTable">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>951</width>
         <height>531</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QLineEdit" name="SearchLineEdit">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>20</y>
       <width>351</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="SearchpushButton">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #00796B;
color: white;
font-family: &quot;Cal Sans&quot;, sans-serif;
font-weight: 500;
font-size: 16px;
border: none;
border-radius: 4px;
padding: 3px 6px;
</string>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QComboBox" name="SearchField">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="RefreshpushButton">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 253);
border: 1px solid #660000; 
border-radius: 6px;
color: #00796B;
font-family: &quot;Cal Sans&quot;, sans-serif;
font-weight: 500;
font-size: 14px;
padding: 3px 6px;
</string>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AddpushButton">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>630</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #388E3C; /* Darker green */
color: white;
font-family: &quot;Cal Sans&quot;, sans-serif;
font-weight: 700; /* Bold */
font-size: 16px;
border: none;
border-radius: 4px;
padding: 6px 12px;
</string>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EditpushButton">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>630</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #1565C0; /* Darker blue */
color: white;
font-family: &quot;Cal Sans&quot;, sans-serif;
font-weight: 700; /* Bold */
font-size: 16px;
border: none;
border-radius: 4px;
padding: 6px 12px;
</string>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DeletepushButton">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>630</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #C62828; /* Darker red */
color: white;
font-family: &quot;Cal Sans&quot;, sans-serif;
font-weight: 700; /* Bold */
font-size: 16px;
border: none;
border-radius: 4px;
padding: 6px 12px;
</string>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>1011</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #800000;  /* Maroon background */
                border: 1px solid #660000;  /* Darker maroon border */
                border-radius: 4px;
                padding: 5px;
                font-family: 'Times New Roman', serif;  /* Formal font */
                font-size: 14px;
                color: white;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 255, 242); /* Soft light green background */
border: 1px solid #660000; /* Darker maroon border */
border-radius: 4px;
padding: 5px;
font-family: 'Roboto', sans-serif;
font-weight: 500; /* Normal weight */
font-style: normal;
font-size: 13px;
color: #800000; /* Maroon text color */
</string>
     </property>
     <property name="text">
      <string>Tenant</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 255, 242); /* Soft light green background */
border: 1px solid #660000; /* Darker maroon border */
border-radius: 4px;
padding: 5px;
font-family: 'Roboto', sans-serif;
font-weight: 500; /* Normal weight */
font-style: normal;
font-size: 13px;
color: #800000; /* Maroon text color */
</string>
     </property>
     <property name="text">
      <string>Room</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 255, 242); /* Soft light green background */
border: 1px solid #660000; /* Darker maroon border */
border-radius: 4px;
padding: 5px;
font-family: 'Roboto', sans-serif;
font-weight: 500; /* Normal weight */
font-style: normal;
font-size: 13px;
color: #800000; /* Maroon text color */
</string>
     </property>
     <property name="text">
      <string>Rent Data</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 255, 242); /* Soft light green background */
border: 1px solid #660000; /* Darker maroon border */
border-radius: 4px;
padding: 5px;
font-family: 'Roboto', sans-serif;
font-weight: 500; /* Normal weight */
font-style: normal;
font-size: 13px;
color: #800000; /* Maroon text color */

</string>
     </property>
     <property name="text">
      <string>Payment</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>10</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 255, 242); /* Soft light green background */
border: 1px solid #660000; /* Darker maroon border */
border-radius: 4px;
padding: 5px;
font-family: 'Roboto', sans-serif;
font-weight: 500; /* Normal weight */
font-style: normal;
font-size: 13px;
color: #800000; /* Maroon text color */

</string>
     </property>
     <property name="text">
      <string>Emergency Record</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>10</y>
       <width>401</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 255, 242); /* Soft light green background */
border: 1px solid #660000; /* Darker maroon border */
border-radius: 4px;
padding: 5px;
font-family: 'Cal Sans', sans-serif;
font-weight: 600;
font-style: normal;
font-size: 16px;
color: #800000; /* Maroon text color */

</string>
     </property>
     <property name="text">
      <string>Sistore Boarding House Management System</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1065</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
